templates/template-creation-web/package.json

```diff
@@ -5,7 +5,8 @@
   "type": "module",
   "scripts": {
     "build": "tsc && vite build",
-    "dev": "vite"
+    "dev": "vite",
+    "start": "vite"
```

Comment:
It's nice to have this in all the templates

Reply:
yeah if you want to do a pr to add this to all the templates I'm ok with adding it in